src/Plane/PlaneFileGlue.hpp

```diff
@@ -37,5 +38,7 @@ namespace PlaneGlue
 /**
  * Throws exception on error.
  */
-void WriteFile(const Plane &plane, Path path);
+  void WriteFile(const Plane &plane, Path path);
+  void WriteFile(const Plane &plane);
```

Comment:
bad indent

Reply:
Sure, to be aligned in PR for plane masses